x-pack/plugins/cloud_security_posture/kibana.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "csp",
```

Comment:
Nit: 

We already have some code in Core that refers to "csp" (that is related to Content-Security-Policy).
As I mentioned in https://github.com/elastic/kibana/pull/124816#discussion_r813544512, your plugin's ID determines how your config settings are scoped, along with several other things such as server logs and feature privileges.

I can see this becoming a source of confusion in the future:
* A user gets some error related to CSP, contacts our support engineers, and your team winds up getting pinged with noise
* Onboarding new team members becomes more difficult because there is other "CSP"-related code in Core that isn't owned by your team
* Etc.

It would be great if you can avoid using the "csp" abbreviation here and elsewhere in your plugin, and instead use "cloudSecurityPosture" to disambiguate.

Reply:
Thanks @jportner , this makes sense. I changed the plugin ID so the settings will be clearly scoped. In the future our team will change file names and constants in a big sweep to avoid any further ambiguity.